package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "general-store",
-  "version": "4.1.0",
+  "version": "4.1.1",
```

```suggestion
  "version": "4.1.0",
```
this can stay as 4.1.0, i'll use `npm version patch` post-merge to bump the version so it pushes a tag to github as well